README.md

```diff
@@ -10,21 +10,21 @@ Go back to the start of time, where DataStore's weren't complex.
 
 ## Installation
 
-Start off by copying [`greenwich.lua`](/greenwich.lua) over to `ServerStorage`. Make sure it a ModuleScript named `Greenwich`.
+Start off by copying [`greenwich.lua`](/greenwich.lua) over to `ServerStorage`. Make sure its a ModuleScript named `Greenwich`.
```

Comment:
It should be `it's`..

Reply:
no way you just grammar corrected me on a grammar correction pr -_-